packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -102,6 +106,11 @@ export const UserSelect: ButtonStory = {
   }
 };
 
+export const IsPending: ButtonStory = {
+  name: 'Pending spinner',
+  render: (args) => <Pending {...args} />
```

Comment:
nit pick, render first, name second
you could also get rid of name if you called the exported const `PendingSpinner` instead